fairroot/online/source/FairLmdSource.cxx

```diff
@@ -236,7 +241,17 @@ Int_t FairLmdSource::ReadEvent(UInt_t)
 
 void FairLmdSource::Close()
 {
+    CloseLmd();
+}
```

Comment:
This should be implemented in the base class `FairSource`:
```cpp
void FairSource::Close() {Close_imp();}
```

Reply:
Can we postpone this for another PR?